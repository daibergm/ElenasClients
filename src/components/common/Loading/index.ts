export { default as Loading } from './LoadingComponent';
