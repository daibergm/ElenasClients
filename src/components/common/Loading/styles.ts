import { StyleSheet } from 'react-native';
import { Colors } from '../../../assets';

export default StyleSheet.create({
  container: {
    position: 'absolute',
    backgroundColor: Colors.white,
    width: '100%',
    height: '100%',
    justifyContent: 'center',
  },
});
