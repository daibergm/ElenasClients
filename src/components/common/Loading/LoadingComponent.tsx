import React from 'react';
import { View, ActivityIndicator } from 'react-native';

// @Styles
import styles from './styles';
import { Colors } from '../../../assets';

function LoadingComponent() {
  return (
    <View style={styles.container}>
      <ActivityIndicator size="large" color={Colors.primary} />
    </View>
  );
}

export default LoadingComponent;
