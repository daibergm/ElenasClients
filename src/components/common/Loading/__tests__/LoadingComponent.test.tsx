import React from 'react';
import { render } from '@testing-library/react-native';

// @Components
import Loading from '../LoadingComponent';

describe('Loading test suits', () => {
  it('Render correctly', () => {
    const loading = render(<Loading />);
    expect(loading).toMatchSnapshot();
  });
});
