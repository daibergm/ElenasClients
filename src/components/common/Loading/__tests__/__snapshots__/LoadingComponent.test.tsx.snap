// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading test suits Render correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "height": "100%",
      "justifyContent": "center",
      "position": "absolute",
      "width": "100%",
    }
  }
>
  <ActivityIndicator
    color="#00adee"
    size="large"
  />
</View>
`;
